src/components/dashboard/Dashboard.js

```diff
@@ -3,12 +3,15 @@ import './Dashboard.css';
 import FullCalendar from '@fullcalendar/react'
 import listPlugin from '@fullcalendar/list'
 import {ListWrapper} from './ListWrapper'
+import TourDisplay from './TourDisplay'
+import ContactsPreview from './ContactsPreview'
 
 
 const Dashboard = () => {
   return (
     <div className="Dashboard">
-      <h1>THIS IS THE Dashboard</h1>
```

Comment:
Nice! Got rid of the "this is dashboard" 😂

Reply:
How will we know?